skiko/src/commonMain/kotlin/org/jetbrains/skia/Matrix22.kt

```diff
@@ -0,0 +1,47 @@
+package org.jetbrains.skia
+
+import org.jetbrains.skia.impl.InteropPointer
+import org.jetbrains.skia.impl.InteropScope
+import org.jetbrains.skia.impl.withResult
+
+/**
+ * 2x2 matrix.
+ */
+class Matrix22(vararg mat: Float) {
+    /**
+     * Matrix elements are in row-major order.
+     */
+    internal val mat: FloatArray
+
+    override fun equals(other: Any?): Boolean {
+        if (other === this) return true
+        if (other !is Matrix22) return false
+        return mat.contentEquals(other.mat)
+    }
+
+    override fun hashCode(): Int {
+        val PRIME = 59
+        var result = 1
+        result = result * PRIME + mat.contentHashCode()
+        return result
+    }
+
+    override fun toString(): String {
+        return "Matrix22(_mat=$mat)"
+    }
+
+    companion object {
```

Comment:
Could you add IDENTITY constant for this matrix?

Reply:
Added